README.md

```diff
@@ -30,7 +30,8 @@ Building The Provider
 Clone repository to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-ns1`
 
 ```sh
-$ mkdir -p $GOPATH/src/github.com/terraform-providers; cd $GOPATH/src/github.com/terraform-providers
+$ mkdir -p $GOPATH/src/github.com/terraform-providers
+$ cd $GOPATH/src/github.com/terraform-providers
```

Comment:
Is creating a home for the provider in $GOPATH necessary anymore, since we're using go mod?

Reply:
i didn't wanna touch it cause i know we still ship a vendor dir around

Reply:
Ah gotcha, yeah I don't see any harm in leaving it as is.  Google's provider README still does it this way so 🤷‍♂️